readme.md

```diff
@@ -115,6 +115,13 @@ Run `brew cask install suspicious-package` or [download manually](http://www.mot
 [![](screenshots/SuspiciousPackage.png)](http://www.mothersruin.com/software/SuspiciousPackage/)
 
 
+### [QLRest](https://github.com/cluther/qlrest)
+
+> Preview ReST (ReStructuredText) files
+
+Run `brew cask install qlrest` or [download manually](https://github.com/downloads/cluther/qlrest/QLRest.qlgenerator.zip)
+
```

Comment:
Sorry for the delay. Had a hard time making that plugin to work. Let's close this PR as I'm afraid I'm not going to crack it. I'll re-open it if I find a way in the future.